src/components/victory-candlestick/candle.js

```diff
@@ -0,0 +1,57 @@
+import React, { PropTypes } from "react";
+// import d3Shape from "d3-shape";
+
+
+export default class Candle extends React.Component {
+  static propTypes = {
+    index: React.PropTypes.number,
+    wickX: PropTypes.number,
+    candleX: PropTypes.number,
```

Comment:
I would prefer if these were `x`, `y1`, `y2` to match with standard svg attr names. Wick vs. Candle x values should be calculated in this component.